chainservice/chainservice.go

```diff
@@ -632,13 +632,13 @@ func (cs *ChainService) ActionPool() actpool.ActPool {
 
 // APIServer defines the interface of core service of the server
 type APIServer interface {
-	ActionByActionHash(h hash.Hash256) (action.SealedEnvelope, error)
-	ReceiptByAction(h hash.Hash256) (*action.Receipt, string, error)
```

Comment:
this interface is used internally, IMO no need to change to protobuf data struct